articles/aks/tutorial-kubernetes-deploy-cluster.md

```diff
@@ -28,7 +28,7 @@ In additional tutorials, the Azure Vote application is deployed to the cluster,
 
 In previous tutorials, a container image was created and uploaded to an Azure Container Registry instance. If you haven't done these steps, and would like to follow along, start at [Tutorial 1 – Create container images][aks-tutorial-prepare-app].
 
-This tutorial requires that you're running the Azure CLI version 2.0.75 or later. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI][azure-cli-install].
+This tutorial requires that you're running the Azure CLI version 2.0.53 or later. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI][azure-cli-install].
```

Comment:
According to all the other steps in this tutorial, this tutorial requires version 2.0.53 of the Azure CLI. This is step is the only place where version of 2.0.75 is mentioned.

- https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-prepare-acr#before-you-begin
- https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-deploy-application#before-you-begin
- https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-scale#before-you-begin
- https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-app-update#before-you-begin
- https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-upgrade-cluster#before-you-begin